python/Ganga/Core/GangaRepository/__init__.py

```diff
@@ -20,3 +20,10 @@ def getRegistries():
 
 def getRegistry(name):
     return allRegistries[name]
+
+def getRegistrySlice(name):
+    return allRegistries[name].getSlice()
+
+def getRegistryProxy(name):
+    return allRegistries[name].getProxy()
+
```

Comment:
Is it worth adding these functions if they're only used for testing? We should aim to keep the API as simple as possible.